src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SENTENCE_SPLITTER = "next sentence";
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private String[] getDataFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                stringBuilder.append(line).append(SENTENCE_SPLITTER);
```

Comment:
you don't need builder here, you can get buy and supply from each line if you split it by separator

Reply:
Мне здесь нужен билдер, так как в этом методе я только считываю данные с файла в массив. Каждая ячейка массива - это полноценная строка из файла в том виде, в котором она там прописана. А проводить манипуляции с этими строками и составлять репорт я уже начинаю в следующем методе.

Reply:
Yes but your approach requires more operations: first you read file line by line and append it to the single string. Then you split it once again and iterate each line to get operations sum. You can avoid second iterattion if you parse values instead of appending them to your stringBuilder. Now you just append lines to split them and pass to another method

Reply:
done.